packages/gatsby/index.d.ts

```diff
@@ -41,7 +48,7 @@ export const prefetchPathname: (path: string) => void
  * export default (props: PageProps) => {
  *
  * @example
- * // When adding types for both pageContext (represended by LocaleLookUpInfo) 
+ * // When adding types for both pageContext (represended by LocaleLookUpInfo)
```

Comment:
```suggestion
 * // When adding types for both pageContext (represented by LocaleLookUpInfo)
```

Might as well fix a typo while we're in here.